src/kibana/components/agg_types/controls/percentiles.html

```diff
@@ -0,0 +1,9 @@
+<div class="form-group">
+  <label>Percents</label>
+  <kbn-number-list
+    ng-model="agg.params.percents"
+    unit-name="percent"
+    range="(0,100)"
```

Comment:
From the [documentation](http://www.elastic.co/guide/en/elasticsearch/reference/1.x/search-aggregations-metrics-percentile-aggregation.html): "Requested percentiles must be a value between 0-100 inclusive." It doesn't seem like I can input 0 or 100 as it currently is.